kata-1/main.tf

```diff
@@ -0,0 +1,80 @@
+resource "aws_vpc" "main" {
+  cidr_block = var.vpc_cidr
+

```

Comment:
it's good to set dns hostnames and resolution within your vpc.
this will enable all your ec2 instances to also have a dns names and the vpc router can resolve these names into private ip's for you.

Reply:
Added on [this](https://github.com/adriaanbd/terraform_aws_dojo/pull/1/commits/7d409be6845ff28a0f8588f4020830c68d99fced) commit